systest/cluster/nodes.go

```diff
@@ -38,16 +39,46 @@ var (
 		fastnet.SmesherConfig,
 	)
 
-	smesherRequests = apiv1.ResourceList{
-		apiv1.ResourceCPU:    resource.MustParse("0.5"),
-		apiv1.ResourceMemory: resource.MustParse("1Gi"),
-	}
-	smesherLimits = apiv1.ResourceList{
-		apiv1.ResourceCPU:    resource.MustParse("4"),
-		apiv1.ResourceMemory: resource.MustParse("1Gi"),
-	}
+	smesherResources = parameters.NewParameter(
+		"smesher_resources",
+		"requests and limits for smesher container",
+		&apiv1.ResourceRequirements{
+			Requests: apiv1.ResourceList{
+				apiv1.ResourceCPU:    resource.MustParse("0.5"),
+				apiv1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+			Limits: apiv1.ResourceList{
+				apiv1.ResourceCPU:    resource.MustParse("2"),
+				apiv1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+		},
+		toResources,
+	)
+	poetResources = parameters.NewParameter(
+		"poet_resources",
+		"requests and limits for poet container",
+		&apiv1.ResourceRequirements{
+			Requests: apiv1.ResourceList{
+				apiv1.ResourceCPU:    resource.MustParse("1"),
+				apiv1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+			Limits: apiv1.ResourceList{
+				apiv1.ResourceCPU:    resource.MustParse("2"),
+				apiv1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+		},
+		toResources,
+	)
 )
 
+func toResources(value string) (*apiv1.ResourceRequirements, error) {
+	var rst apiv1.ResourceRequirements
+	if err := json.Unmarshal([]byte(value), &rst); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal %s into apiv1.ResourceRequirements: %w", value, err)
```

Comment:
NIT: describe only the context, leave the "failed" keyword for logging
```suggestion
		return nil, fmt.Errorf("unmarshaling %s into apiv1.ResourceRequirements: %w", value, err)
```

to avoid logs like: "Failed <something>: failed to <x>: failed to <y> ..."